app/assets/javascripts/discourse/templates/components/d-editor.hbs

```diff
@@ -28,7 +28,8 @@
 
   <div class="d-editor-preview-wrapper {{if forcePreview 'force-preview'}}">
     <div class="d-editor-preview">
-      {{{preview}}}
+      <div class="d-editor-cooked">{{{preview}}}</div>
+      <div class="d-editor-plugin">{{plugin-outlet name="editor-preview"}}</div>
```

Comment:
do we need the extra `d-editor-plugin` div here? 

I think `plugin-outlet name="editor-preview"` will already wrap it in a div... 

Going to merge this in for now, but perhaps a PR to clean this up and unwrap is in order.